src/frontend/src/btc/schedulers/btc-wallet.scheduler.ts

```diff
@@ -49,13 +49,15 @@ export class BtcWalletScheduler implements Scheduler<PostMessageDataRequestBtc>
 				balance: {
 					data: balance,
 					certified: true
-				}
+				},
+				// TODO: Send parsed transactions
+				newTransactions: ''
```

Comment:
Can we rather use `Omit<PostMessageDataResponseWallet, "newTransactions">` until it's implemented. Using `newTransactions: ""` is per sé incorrect so even if not used, I think it's better to now have invalid code ending on `main`.

Reply:
It's not gonna be merged to main but rather to the BTC transactions feature branch. In any case I improved it a bit by mocking `transactions` to an empty array. In the next PR, I'm gonna create a parser util, and after it is done, will start fetching the real data.

Reply:
Better, thanks.